### PR TITLE
Hotfix/instructor-dashboard-data

### PR DESCRIPTION
**Pull request by m1073496**
### What does this PR do?
* Removes all console logs from codebase
* Fixes bug that was preventing correct student data from rendering to UI

### Anything specific the reviewer should look at?
* From the instructor dashboard, when you click on various students and their projects, the scores and comments should now change to accurately reflect that student's feedback.
* There is still a bug that keeps the previous student's project feedback rendered even after clicking on a new student; the UI doesn't update until you select a module and project for the new student --> this will have to be addressed in a future iteration (I'm thinking add more routes to student buttons that correspond to student ids)

Closes # N/A

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -60,12 +58,12 @@ export default {
...
-      payload = { studentMod: i, studentId: this.userId }
-      // payload = { studentMod: i, studentId: 94 }
-      console.log(payload)
-      this.fetchModule(payload)
+    if (!this.instructorAuth) {
+      let payload
+      for (let i = 1; i < 5; i++) {
+        payload = { studentMod: i, studentId: this.userId }
+        this.fetchModule(payload)
+      }
```
Comment (piknikki) [inline src/App.vue:66]: nice! this is great, so we don't have to pass it around as much. Thank you!!

**Review by piknikki — COMMENTED** [review]
Great work! I'm excited to get to the form and bust out some styling on it.

File: src/store/index.js [hunk 2]
```diff
@@ -28,7 +28,14 @@ export default new Vuex.Store({
...
     setAllModules (state, singleModule) {
-      state.allModules.push(singleModule)
+      if (singleModule === []) {
+        state.allModules = singleModule
+      } else if (state.allModules.length === 4) {
+        state.allModules = []
+        state.allModules.push(singleModule)
+      } else {
+        state.allModules.push(singleModule)
+      }
```
Comment (knees4bees) [inline src/store/index.js:38]: Ohhhh good way to fix this

File: src/store/index.js [hunk 3]
```diff
@@ -84,6 +92,10 @@ export default new Vuex.Store({
         context.commit('setCurrentStudent', foundStudent)
       }
     },
+    clearSelected (context, payload) {
+      context.commit('setCurrentProject', {})
+      context.commit('setCurrentModule', {})
+    },
```
Comment (knees4bees) [inline src/store/index.js:98]: Nice

**Review by knees4bees — APPROVED** [review]
Thanks for fixing this, Jessica!

**Pull request merged**